types/express/express-tests.ts

```diff
@@ -71,6 +71,15 @@ namespace express_tests {
         language = req.acceptsLanguages(['en', 'ja']);
         language = req.acceptsLanguages('en', 'ja');
 
+        let existingHeader1 = req.get('existingHeader') as string;
```

Comment:
I don't like this part. The idea of test files is to test types by checking returning types with assumptions, like:
```ts
let header: string | undefined;
header = req.get('existingHeader');
header = req.header('nonExistingHeader');
header = req.headers.existingHeader;
```
So when someone changes the returned type of this method, the test won't pass. With type assertion it will be false positive.